user-guide/Advanced_Functionality/DataMiner_Systems/Adding_a_DMA/Adding_a_DMA_to_a_DMS_running_STaaS.md

```diff
@@ -4,42 +4,14 @@ uid: Adding_a_DMA_to_a_DMS_running_STaaS
 
 # Adding a DataMiner Agent to a DMS running STaaS
 
-Follow these steps below to connect a DMA to an existing DMS running STaaS:
+## Using a copy of the cloud persisted data
```

Comment:
What is the purpose of this subtitle? As there is no other subtitle on the page, it doesn't seem to make much sense right now, so I think it might be better to replace it with a short introduction to the procedure, which could explain what this "persisted data" is and why you need to make a copy of it.

Reply:
We removed one procedure because it wasn't fully validated yet, hence why the subtitle was still there. I've removed it now.